gp-includes/formats/format-json.php

```diff
@@ -106,6 +106,9 @@ public function read_originals_from_file( $file_name ) {
 			}
 
 			$args['translations'] = (array) $value;
+			if ( ! empty( $args['translations'][0] ) && ! is_string( $args['translations'][0] ) ) {
```

Comment:
I think it should be:
```suggestion
			if ( empty( $args['translations'][0] ) || ! is_string( $args['translations'][0] ) ) {
```
Otherwise an empty array would make it through and would cause the fatal error.